lib/ref_arch_setup/install.rb

```diff
@@ -35,17 +35,54 @@ def initialize(target_master)
     # @param [string] pe_conf_path Path to pe.conf
     # @param [string] pe_tarball Path or URL for the pe tarball
     #
+    # @raise [RuntimeError] Unless either a pe_tarball or pe_version is specified
+    # @raise [RuntimeError] If a valid pe_tarball is not available
+    # @raise [RuntimeError] If a RAS working directory can not be created
```

Comment:
Since you are calling bolt, could get a bolt command error... but somewhere we need to draw the line on over documenting.  This might be a spot it isn't needed.

Reply:
Added @raise [BoltCommandError] since we are calling bolt_helper directly